src/Table.luna

```diff
@@ -152,21 +395,78 @@ class Table:
     def sort sortBy:
         sortByColumns = sortBy.each (colName, order, pos): (self.column colName, order, pos)
         self.sortByColumn sortByColumns
+
+    # Remove rows with missing values.
```

Comment:
Please keep the mood consistent. I prefer the imperative but we keep using indicative throught these docs.

This applies to all of the headers.